styles/components/_button.scss

```diff
@@ -7,18 +7,16 @@ button {
   color: #707070;
 }
 
-.nav-call {
-  button {
-    padding: 9px 32px;
-    font-size: 20px;
-    background: linear-gradient(225deg, transparent 5px, white 0);
-  }
+.button {
+  @extend %button;
+  font-size: 26px;
```

Comment:
Adding CSS after an extend will overwrite any of the values coming in from the extend

For example:
%button {
  font-size: 10px;
}

.button--small {
  @extend %button;
  font-size: 8px;
}


In this case, .button--small will have a font-size of 8px